src/main/java/org/sagebionetworks/web/client/widget/discussion/DiscussionThreadWidget.java

```diff
@@ -59,12 +60,17 @@ public Widget asWidget() {
 	public void configure(DiscussionThreadBundle bundle) {
 		view.clear();
```

Comment:
Since author is not dynamic, just inject a UserBadge in the constructor, and add it to the view.